packages/expo-constants/android/src/main/java/expo/modules/constants/ConstantsService.kt

```diff
@@ -76,7 +77,7 @@ open class ConstantsService(private val context: Context) : InternalModule, Cons
 
   override fun getDeviceYearClass() = YearClass.get(context)
 
-  override fun getIsDevice() = !isRunningOnGenymotion && !isRunningOnStockEmulator
+  override fun getIsDevice() = !isRunningOnEmulator
```

Comment:
```suggestion
  override fun getIsDevice() = !EmulatorUtilities.isRunningOnEmulator()
```

same reason for java bytecode size saving